controllers/k8ssandra/medusa_reconciler_test.go

```diff
@@ -93,6 +94,26 @@ func createMultiDcClusterWithMedusa(t *testing.T, ctx context.Context, f *framew
 	dc1Key := framework.ClusterKey{NamespacedName: types.NamespacedName{Namespace: namespace, Name: "dc1"}, K8sContext: f.DataPlaneContexts[0]}
 	require.Eventually(f.DatacenterExists(ctx, dc1Key), timeout, interval)
 
+	t.Log("check that the standalone Medusa deployment was created")
```

Comment:
It might be good to check that the pod becomes ready here as well as just being created.